defold-rive/editor/src/rive.clj

```diff
@@ -484,7 +616,19 @@
             (dynamic edit-type (g/constantly {:type resource/Resource :ext rive-file-ext}))
             (dynamic error (g/fnk [_node-id rive-file]
                                   (validate-rivescene-riv-file _node-id rive-file))))
-
+           
+  (property material resource/Resource
+            (value (gu/passthrough material-resource))
+            (set (fn [evaluation-context self old-value new-value]
+                   (project/resource-setter evaluation-context self old-value new-value
+                                            [:shader :material-shader]
+                                            [:samplers :material-samplers])))
+            (dynamic edit-type (g/constantly {:type resource/Resource :ext "material"}))
+            (dynamic visible (g/constantly false)))
```

Comment:
In order to use rive.material for proper rendering, but it's not actually part of the data, we make this property not visible.